### PR TITLE
ztp:Add Ptp operator config to enable fast event

### PR DESCRIPTION
**Pull request by aneeshkp**
Signed-off-by: Aneesh Puttur <aneeshputtur@gmail.com>
Adding PTP Operator Config policy to enable or disable fast events.

### CODE REVIEW

File: ztp/source-crs/PtpOperatorConfigForEvent.yaml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+apiVersion: ptp.openshift.io/v1
+kind: PtpOperatorConfig
+metadata:
+  name: default
```
Comment (imiller0) [inline ztp/source-crs/PtpOperatorConfigForEvent.yaml:4]: PtpOperatorConfig is a Namespace scoped CR. Can you add
```suggestion
  name: default
  namespace: openshift-ptp
```

**Review by imiller0 — CHANGES_REQUESTED** [review]

File: ztp/source-crs/PtpOperatorConfigForEvent.yaml (hunk 1 above)
Comment (aneeshkp) [reply ztp/source-crs/PtpOperatorConfigForEvent.yaml:4]: done

**Pull request merged**